tracing/tests/enabled.rs

```diff
@@ -0,0 +1,30 @@
+// liballoc is required because the test subscriber cannot be constructed
+// statically
+#![cfg(feature = "alloc")]
+
+mod support;
+
+use self::support::*;
+use tracing::Level;
+
+#[cfg_attr(target_arch = "wasm32", wasm_bindgen_test::wasm_bindgen_test)]
+#[test]
+fn level_and_target() {
+    let (collector, handle) = collector::mock()
```

Comment:
this is using the v0.2 trait names...this test _shouldn't_ compile, since the `collector` module doesn't exist. if this didn't cause a compiler error, that's a problem...
```suggestion
    let (subscriber, handle) = subscriber::mock()
```

Reply:
yah....i cant figure out how to get this to actually run...will look into it

Reply:
i figured out why the test was not failing to compile as i would have expected it to: https://github.com/tokio-rs/tracing/pull/1883/files#r797144693